### PR TITLE
Windows 7 EOL Docs Update

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

## Description
With EOL for Windows 7 and Windows 2008 happening this month, related documentation needs updating.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,7 +332,6 @@ In addition, runtime support is provided for:
 | Debian   | 8, 9     | x86_64 |
 | RHEL     | 6, 7     | x86_64 |
 | Ubuntu   | 12.04+   | x86_64 |
-| Windows  | 7+       | x86_64 |
```
Comment (mjingle) [inline README.md:335]: Should this be changed to `8+` rather than removing this Windows row?

File: README.md [hunk 2]
```diff
@@ -312,7 +312,7 @@ Remote Targets
 | Oracle Enterprise Linux      | 5, 6, 7                                          | i386, x86_64  |
 | Red Hat Enterprise Linux     | 5, 6, 7                                          | i386, x86_64  |
 | Solaris                      | 10, 11                                           | sparc, x86    |
-| Windows\*                    | 7, 8, 8.1, 10, 2008, 2008R2 , 2012, 2012R2, 2016 | x86, x86_64   |
+| Windows\*                    | 8, 8.1, 10, 2008, 2008R2, 2012, 2012R2, 2016     | x86, x86_64   |
```
Comment (zenspider) [inline README.md]: OH LOOK A TABLE! :P

**Review by zenspider — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mjingle) [reply README.md]: But it's on the readme and not the website! 😂

File: README.md (hunk 1 above)
Comment (mjingle) [reply README.md:335]: @zenspider ^ ?

Comment (clintoncwolfe) [reply README.md:335]: I would think so.

**Review by clintoncwolfe — APPROVED** [review]
Generally looks good, let's add that 8+ back in and I think we're all set - thanks!

**Pull request merged**